src/+otp/RHS.m

```diff
@@ -39,6 +39,12 @@
                 f = fields{i};
                 obj.(f) = extras.(f);
             end
+            
+            % OCTAVE FIX: ode15s throws an error when the Vectorized option is set. It seems no
+            % integrator uses this option anyway.
+            if otp.utils.compatibility.isOctave()
+                obj.Vectorized = [];
```

Comment:
In octave or in matlab? No integrator uses this option?

Reply:
Octave only